test/Runtime/POSIX/MixedConcreteSymbolic.c

```diff
@@ -0,0 +1,22 @@
+// RUN: %llvmgcc %s -emit-llvm -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out -allow-external-sym-calls --exit-on-error --libc=uclibc --posix-runtime %t.bc --sym-stdin 10  2>%t.log > %t.output
+// RUN: grep a[a-z] %t.output
```

Comment:
Please change to `FileCheck` and the pattern is wrong - it should be `a[b-y]`.

Reply:
Good point, cheers!